### DISCUSSION TITLE
Configuring GraphIQL Call to /graphql Endpoint

### DISCUSSION THREAD

**Discussion by jahhuang** [Netflix/dgs-framework#1731] [Q&A]
Hi all,

In my `application.properties` file, I have set the path for the graphql and graphiql endpoints like so:
```
dgs.graphql.path=/customPrefix/graphql
dgs.graphql.graphiql.path=/customPrefix/graphiql
```

I'm able to access the Graphiql interface: http://localhost:8080/customPrefix/graphiql

When using Graphiql, the network POST request from the interface to the service is: http://localhost:8080/customPrefix/graphql

My service sits behind a proxy, so the call to the /graphql endpoint sits behind another URL prefix. Is there a way to configure this call path to the service without changing the graphiql path? In particular, I need to add another prefix and add query parameters to the URL, i.e. something like http://localhost:8080/anotherPath/customPrefix/graphql?param1=value1&param2=value2

I saw this issue: https://github.com/Netflix/dgs-framework/issues/171, but it doesn't quite match what I want to do here.

Thanks in advance!

**Comment by srinivasankavitha** [comment]
Beyond the 2 config properties you listed above, we don't have any other knobs to configure the path. I don't think it would work with adding query parameters in any case.
